appinventor/components/src/com/google/appinventor/components/runtime/Web.java

```diff
@@ -325,6 +331,31 @@ public void ResponseFileName(String responseFileName) {
     this.responseFileName = responseFileName;
   }
 
+  /**
+   * Returns the number of milliseconds that each request will wait for a response before they time out.
+   * If set to 0, then the request will wait for a response indefinitely.
+   */
+  @SimpleProperty(category = PropertyCategory.BEHAVIOR,
+      description = "The number of milliseconds that a web request will wait for a response before giving up." +
+              "If set to 0, then there is no time limit on how long the request will wait.")
```

Comment:
Indentation here should be +4 from the `d` in `description`.

Reply:
Fixed this